app/models/incoming_message.rb

```diff
@@ -55,7 +55,8 @@ class IncomingMessage < ActiveRecord::Base
   # never really has many info_request_events, but could in theory
   has_many :info_request_events, :dependent => :destroy
 
-  belongs_to :raw_email, :dependent => :destroy
```

Comment:
possible monkeypatch here if the original fix proves problematic https://gist.github.com/lizconlan/7e577eeff16ad5a83b943601d4241aee